src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs

```diff
@@ -80,22 +80,26 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var succeeded = false;
             if (helper.CanOfferUseExpressionBody(optionSet, declaration, forAnalyzer: false))
             {
-                context.RegisterRefactoring(new MyCodeAction(
-                    helper.UseExpressionBodyTitle.ToString(),
-                    c => UpdateDocumentAsync(
-                        document, root, declaration, optionSet, helper,
-                        useExpressionBody: true, cancellationToken: c)));
+                context.RegisterRefactoring(
+                    new MyCodeAction(
+                        helper.UseExpressionBodyTitle.ToString(),
+                        c => UpdateDocumentAsync(
+                            document, root, declaration, optionSet, helper,
+                            useExpressionBody: true, cancellationToken: c)),
+                        declaration.Span);
```

Comment:
indentation seems wrong.

Reply:
Nope, it's correct here.  Or at least as correct as it can be, IMO. Or what exactly is wrong here?

Reply:
the span is a sibling arg to the code action.  so i would expect this aligned with "new MyCodeAction" right?

Reply:
Ah, right.